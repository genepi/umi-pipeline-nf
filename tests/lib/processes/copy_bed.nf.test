nextflow_process {

    name "Test Process COPY_BED"
    script "lib/processes/copy_bed.nf"
    process "COPY_BED"

    test("Should run without failures") {

        when {
            process {
                """
                input[0] = file("$baseDir/tests/input/pipeline/ref/lpa-ref2645.bed", checkIfExists: true)
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
