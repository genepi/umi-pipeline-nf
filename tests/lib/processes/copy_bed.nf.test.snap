{
    "Should run without failures": {
        "content": [
            {
                "0": [
                    "lpa-ref2645.bed:md5,6291e11874973cee03800a3bb45345fc"
                ],
                "bed": [
                    "lpa-ref2645.bed:md5,6291e11874973cee03800a3bb45345fc"
                ]
            }
        ],
        "meta": {
            "nf-test": "0.8.4",
            "nextflow": "23.10.0"
        },
        "timestamp": "2024-03-07T15:17:45.33752"
    }
}